frontend/src/framework/notification/enums/notification-type.enums.ts

```diff
@@ -0,0 +1,9 @@
+const enum NotificationType {
+    INFO = 'info',
+    SUCCESS = 'success',
+    WARNING = 'warning',
+    ERROR = 'error',
+    DEFAULT = 'default',
+}
```

Comment:
```suggestion
const NotificationType = {
    INFO: 'info',
    ...
} as const
```
We chose to use 'as const' enums instead of TypeScript enums everywhere.

PS: everything else LGTM

Reply:
Thanks, I understood